app/models/appointment_summary.rb

```diff
@@ -32,7 +32,8 @@ def postcode=(str)
   has_many :appointment_summaries_batches, dependent: :destroy
 
   validates :title, presence: true, inclusion: { in: TITLES, allow_blank: true }
-  validates :last_name, presence: true
+  validates :first_name, length: { maximum: 50 }
+  validates :last_name, presence: true, length: { maximum: 50 }
```

Comment:
I think the desired behaviour is that the combined `title`, `first_name` and `last_name` can't be greater than 50 characters. This is so the full name can be seen when shown through a C5 envelope window.